AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_entity_utils.py

```diff
@@ -313,6 +313,42 @@ def get_type_ids(component_names: list, entity_type: EditorEntityType = EditorEn
             bus.Broadcast, "FindComponentTypeIdsByEntityType", component_names, entity_type.value)
         return type_ids
 
+    def is_property_visible(self, component_property_path: str, expected: bool = True) -> bool:
+        """
+        Used to determine if a property is visible.
+        param component_property_path: String of component property. (e.g. 'Settings|Visible')
+        :param expected: The expected visibility (default True)
+        :return: None
+        """
+        visible_key = 1
```

Comment:
question: I'm not quite sure where `visible_key` comes from here or why it is this value. Is it a lookup into a map/dictionary or an array offset?

Reply:
This is buried in `EditorEntity.get_property_type_visibility()`.

```
"""
        Used to work with Property Tree Editor build_paths_list_with_types.
        Some component properties have unclear type or return to much information to contain within one Report.info.
        The iterable dictionary can be used to print each property path and type individually with a for loop.
        :return: Dictionary where key is property path and value is a tuple (property az_type, UI visible)
        """
```

I'm open to a better pattern here to eliminate the mystery meat variable of just putting a 1 into retrieving the value.

Reply:
Hmm maybe just a comment explaining the context of this would be useful?